### PR TITLE
build(lidar_centerpoint_tvm): remove artifacts download

### PR DESCRIPTION
**Pull request by lexavtanke**
## Description

See https://github.com/autowarefoundation/autoware.universe/issues/3137
This PR updates lidar_centerpoint_tvm to use updated version `get_neural_network` function from tvm_utility package introduced in https://github.com/autowarefoundation/autoware.universe/pull/4923. 
It also uses artifacts located in autoware_data folder as introduced in https://github.com/autowarefoundation/autoware/pull/3916

## Tests performed

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by xmfcx** [comment]
Thanks for this PR!

Could you resolve the conflicts for the `.cspell-partial.json` file?

File: perception/lidar_centerpoint_tvm/include/lidar_centerpoint_tvm/centerpoint_tvm.hpp
Comment (ambroise-arm) [inline perception/lidar_centerpoint_tvm/include/lidar_centerpoint_tvm/centerpoint_tvm.hpp:63]: Please use the same position you placed `data_path` at for `tvm_utility::pipeline::InferenceEngineTVM` (after `pkg_name`)

Comment (lexavtanke) [reply perception/lidar_centerpoint_tvm/include/lidar_centerpoint_tvm/centerpoint_tvm.hpp]: I updated the code. Could you please review it again?

**Review by ambroise-arm — APPROVED** [review]

**Comment by lexavtanke** [comment]
@angry-crab Could you please approve the changes when you have time? You are the code owner of this package. @ambroise-arm has reviewed and approved the changes, but they won't merge unless a code owner also approves it.
Thank you in advance!

**Pull request merged**
